pkg/crypto/encryption_test.go

```diff
@@ -45,9 +45,8 @@ func TestEncryptDecrypt(t *testing.T) {
 	} {
 		t.Run(fmt.Sprintf("%s", tc.Name), func(t *testing.T) {
 			a := assertions.New(t)
-			rand := random.New()
 			var key types.AES128Key
-			err := key.UnmarshalBinary(rand.Bytes(16))
+			err := key.UnmarshalBinary(random.Bytes(16))
```

Comment:
Shouldn't we make this consistent with the other files?

```suggestion
			rand.Read(key[:])
```